configurations/manager/debian9.yaml

```diff
@@ -0,0 +1,4 @@
+ami_id_monitor: 'ami-0cfac3931b2a799d1'  # Debian stretch image
```

Comment:
can you this configuration file ? it's not related those changes

Reply:
this whole file was marked as "deleted by us", so i "returned" it...
maybe this branch doesn't need it??

Reply:
"deleted by us" means that your cherry-pick was dependant on the file that doesn't exist in the branch you are backporting to, so it notifies that it was deleted. No need to add it.